bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/net/NetUtil.java

```diff
@@ -80,6 +122,20 @@ public synchronized void modified(Map<String, Object> config) {
         } else {
             configuredBroadcastAddress = broadcastAddressConf;
         }
+
+        String pollFrequenceSecondsStr = "";
+        int pollFrequenceSeconds = POLL_DEFAULT_FREQUENCY_SECONDS;
```

Comment:
rename to `pollIntervalSeconds`, same for `pollFrequenceSeconds`.

Reply:
addressed *interval*